classes/class-wc-connect-jetpack-adapter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Automattic\Jetpack\Connection\Tokens;
+use Automattic\Jetpack\Connection\Manager;
+
+class WC_Connect_Jetpack_Adapter {
+	public static function get_master_user_access_token( $user_id ) {
+		if ( class_exists( '\Automattic\Jetpack\Connection\Manager' ) ) {
```

Comment:
Since this check needed to be made in multiple places, I created a new separate class to localize it.